src/pages/ContentScripts/features/developer-hovercard-info/view.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import getGithubTheme from '../../../../helpers/get-github-theme';
+import optionsStorage, { HypercrxOptions, defaults } from '../../../../options-storage';
+import { useTranslation } from 'react-i18next';
+import '../../../../helpers/i18n';
+import { rocketLight, rocketDark } from './base64';
+
+interface OpenRankProps {
+  developerName: string;
+  openrank: string;
+}
+
+const OpenRankView: React.FC<OpenRankProps> = ({ developerName, openrank }) => {
+  const [theme, setTheme] = useState<'light' | 'dark'>(getGithubTheme() as 'light' | 'dark');
+  const [options, setOptions] = useState<HypercrxOptions>(defaults);
+  const { t, i18n } = useTranslation();
+  useEffect(() => {
+    (async function () {
+      setOptions(await optionsStorage.getAll());
+      i18n.changeLanguage(options.locale);
+    })();
+  }, [options.locale]);
+
+  useEffect(() => {
+    const handleThemeChange = () => {
+      const newTheme = getGithubTheme() as 'light' | 'dark';
+      setTheme(newTheme);
+    };
+
+    // Listen for theme change events
+    window.addEventListener('themeChange', handleThemeChange);
+
+    return () => {
+      window.removeEventListener('themeChange', handleThemeChange);
+    };
+  }, []);
+
+  return (
+    <div className={`openrank-info-container ${theme}`} data-developer-name={developerName}>
+      <div className="openrank-info" style={{ color: theme === 'light' ? 'black' : 'white' }}>
+        <img
+          width={20}
+          height={20}
+          style={{ display: 'inline-block', verticalAlign: 'middle', position: 'relative', left: '-2.5px' }}
+          src={theme === 'light' ? rocketLight : rocketDark}
+          alt=""
+        />
+        <span style={{ display: 'inline-block', verticalAlign: 'middle' }}>OpenRank {openrank}</span>
```

Comment:
Can we use the similar style for the other lines. I think you can use their class directly

<img width="365" alt="image" src="https://github.com/user-attachments/assets/e7c4650d-83ed-44c8-8736-e4831bf89369">

Reply:
I think this display effect highlights the work of hypercrx, and it is easy to be ignored if it is integrated. I will deal with other problems first, and if you think it really needs to be changed, I will modify it.

Reply:
I agree with highlighting it. But we should use the same color for other font in the project. I think it is `color-text-secondary`. Can you take a look?

Reply:
OK i will modify it later

Reply:
The effect cannot be changed by directly using the class, so I achieved the target effect by defining the font color of two modes.
![QQ_1721531387486](https://github.com/user-attachments/assets/6fe0f254-796d-43ee-aa2f-64be97bd7b38)
![QQ_1721531409539](https://github.com/user-attachments/assets/84de6899-0c1c-4c05-a507-ddd30dc14a08)